Require/RequireJsonCorruptFile/index.js

```diff
@@ -1,5 +1,5 @@
 try {
     require('./test.json');
 } catch (e) {
-    TNSLog(e.message);
+	TNSLog("There was a corrupt or invalid package.json.");
```

Comment:
although it looks wierd here it's one tab, it should look fine when you download it localy